### PR TITLE
Handle error in overview and tagcloud context

### PR DESCRIPTION
**Pull request by anishsarangi**
In this PR I have handled many edge cases during API calls in overview context

### CODE REVIEW

File: scripts/overview.js [hunk 1]
```diff
@@ -100,18 +108,14 @@ function get_thumbnail () {
...
-      }
+      $('#show_thumbnail').append($('<img>').attr('src', new_url))
     })
     .catch(function (exception) {
       $('#loader_thumbnail').hide()
       if (exception === 'timeout') {
         $('#show_thumbnail').text('Please refresh the page...Time out!!')
       } else {
-        if (response.status === 504) { // TODO: FIXME: response not defined
+        if (exception.status === 504) { // TODO: FIXME: response not defined
```
Comment (tikhsuP) [inline scripts/overview.js:118]: The code looks good.
Can we remove this 'TODO' comment?

Comment (anishsarangi) [reply scripts/overview.js:118]: Yes actually, Still I am not sure of what is the exact response when the API call is failing or getting success !! For now we can delete it !

Comment (tikhsuP) [reply scripts/overview.js]: Okay, thanks :-)

File: scripts/overview.js [hunk 2]
```diff
@@ -63,25 +69,26 @@ function timestamp2datetime (timestamp) {
...
     } else {
       const ts = data[1][1]
       const dt = timestamp2datetime(ts).toString().split('+')[0]
       $('#first_archive_datetime')
         .text(dt)
-    		.attr('href', 'https://web.archive.org/web/' + ts + '/' + url)
+        .attr('href', 'https://web.archive.org/web/' + ts + '/' + url)
     }
   })
+  .fail(() => $('#first_archive_datetime_error').text('Data are not available'))
```
Comment (cgorringe) [inline scripts/overview.js:84]: Just a minor nitpick: 'Data not available' reads better.

**Comment by cgorringe** [comment]
Somehow can't "approve" now, so I approve this code...

**Comment by anishsarangi** [comment]
@cgorringe I have pushed the required changes !

**Pull request merged**